test/parallel/test-async-wrap-check-providers.js

```diff
@@ -114,20 +114,30 @@ new ChildProcess();
 
 new HTTPParser(HTTPParser.REQUEST);
 
+{
+  const server = http2.createServer();
+  server.on('stream', common.mustCall((stream) => {
+    stream.respond({ ':status': 200 });
+    stream.end('hello world');
+  }));
+  server.listen(0, () => {
+   const client = http2.connect(`http://localhost:${server.address().port}`);
+   const req = client.request({':path': '/'});
+   req.on('response', common.mustCall());
+   req.on('data', common.noop);
+   req.on('end', common.mustCall(() => {
```

Comment:
Yeah. Just pointing out that the test will ultimately fail anyway if server.listen isn't called.